files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/destinations.yml

```diff
@@ -117,122 +88,127 @@ destinations:
       require:
         - mira-pulsar
       accept:
         - docker
-        - condor-singularity-with-conda-python
         - condor-tpv
         - singularity
 
   pulsar_sanjay_tpv:
+    inherits: pulsar_default
     runner: pulsar_sanjay_runner
     max_accepted_cores: 8
     max_accepted_mem: 15
-    inherits: pulsar_default
     scheduling:
       require:
         - sanjay-pulsar
       accept:
         - docker
-        - condor-singularity-with-conda-python
         - condor-tpv
         - singularity
 
   pulsar_sk01_tpv:
-    runner: pulsar_eu_sk01
     inherits: pulsar_default
+    runner: pulsar_eu_sk01
     max_accepted_cores: 8
     max_accepted_mem: 16
     scheduling:
       require:
         - sk-pulsar
       accept:
         - docker
-        - condor-singularity-with-conda-python
         - condor-tpv
         - singularity
 
   pulsar_it_tpv:
-    runner: pulsar_eu_it01
     inherits: pulsar_default
+    runner: pulsar_eu_it01
     max_accepted_cores: 16
     max_accepted_mem: 31
 
-
   pulsar_be_tpv:
-    runner: pulsar_eu_be01
     inherits: pulsar_default
+    runner: pulsar_eu_be01
     max_accepted_cores: 8
     max_accepted_mem: 15
 
-  condor_singularity_with_conda_python:
+  #############################
+  # LOCAL CONDOR DESTINATIONS #
+  #############################
+
+  condor_singularity:
+    inherits: basic_singularity_destination
+    runner: condor
+    max_accepted_cores: 24
+    max_accepted_mem: 128
+    params:
+      metadata_strategy: "extended"
+    scheduling:
+      prefer:
+        - singularity
+
+  # Generic destination for tools that don't get any params
+  # and no specified dependency resolution
+  condor_tpv:
     runner: condor
     max_accepted_cores: 64
     max_accepted_mem: 1000
     scheduling:
-      require:
-        - condor-singularity-with-conda-python
+      prefer:
+        - condor-tpv
+
+  condor_singularity_with_conda:
+    inherits: basic_singularity_destination
+    runner: condor
+    max_accepted_cores: 64
+    max_accepted_mem: 1000
     params:
-      singularity_enabled: true
-      # can and should be made stricter at some point
-      singularity_volumes: "$_CONDOR_SCRATCH_DIR:rw,$job_directory:rw,$tool_directory:ro,$job_directory/outputs:rw,$working_directory:rw,/data/db/:ro,/data/dnb01/galaxy_db/:ro,/data/dnb02/galaxy_db/:ro,/data/dnb03/galaxy_db/:ro,/data/dnb05/galaxy_db/:ro,/data/dnb06/galaxy_db/:rw,/data/dnb07/galaxy_db/:rw,/data/dnb08/galaxy_db/:rw,/data/dp01/galaxy_db/:rw,/data/0/galaxy_db/:ro,/data/1/galaxy_db/:ro,/data/2/galaxy_db/:ro,/data/3/galaxy_db/:ro,/data/4/galaxy_db/:ro,/data/5/galaxy_import/galaxy_user_data/:ro,/data/6/galaxy_db/:ro,/data/7/galaxy_db/:ro,/usr/local/tools/:ro"
       container_override:
         - type: singularity
           shell: "/bin/bash"
           resolve_dependencies: true
           identifier: "/data/0/singularity_base_images/centos:8.3.2011"
+    scheduling:
+      require:
+        - singularity
+        - conda
```

Comment:
Does that mean that all tools needs to have a tag=conda?

Reply:
yes because it is a destination that is only used by tools that want to run conda inside singularity